test/libsolidity/GasCosts.cpp

```diff
@@ -69,33 +79,35 @@ BOOST_AUTO_TEST_CASE(string_storage)
 	m_compiler.overwriteReleaseFlag(true);
 	compileAndRun(sourceCode);
 
+	u256 bzzr0Cost = GasMeter::dataGas(dev::swarmHash(m_compiler.metadata("C")).asBytes(), true);
```

Comment:
Maybe put that into the macro? It may be a good idea anyway to have a macro for deployment gas and a macro for runtime gas.

Reply:
Right - I didn't want to have to calculate the hash multiple times, but it's only done once anyways of course! I'll do that once the current test run verifies that the fix generally works.